virt/live_migration/virt-configuring-live-migration.adoc

```diff
@@ -10,24 +10,14 @@ You can configure live migration settings to ensure that the migration processes
 
 You can configure live migration policies to apply different migration configurations to groups of virtual machines (VMs).
 
-[id="live-migration-settings"]
-== Live migration settings
-
-You can configure the following live migration settings:
-
-* xref:../../virt/live_migration/virt-configuring-live-migration.adoc#virt-configuring-live-migration-limits_virt-configuring-live-migration[Limits and timeouts]
-* xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[Maximum number of migrations per node or cluster]
-
-include::modules/virt-configuring-live-migration-limits.adoc[leveloffset=+2]
```

Comment:
Could this file be removed from the repo now or moved to `_unused_topics/`? Not seeing it included anywhere else.